src/deployment.h

```diff
@@ -70,6 +70,13 @@ inline bool createQtConf(appdir::AppDir &appDir) {
 }
 
 inline bool createAppRunHook(appdir::AppDir &appDir) {
+    auto themesPath = appDir.path() / "usr/plugins/platformthemes";
+
+    if (!bf::exists(themesPath) || bf::is_empty(themesPath)) {
```

Comment:
I don't think we need this check, it should be safe to add the script in any case. Why did you add it? Just curious.

Reply:
Well, having the hook is useless without the theme plugins, and since it's an easy check it seemed like a good idea. I'll remove it.